engine/src/main/java/com/redhat/cloud/notifications/templates/models/ServerConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.cloud.notifications.templates.models;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class ServerConfig {
+
+    @ConfigProperty(name = "env", defaultValue = "local-dev")
```

Comment:
Shouldn't this be
```suggestion
    @ConfigProperty(name = "env.name", defaultValue = "local-dev")
```
to match the existing `ENV_NAME` environment variable that is already used on stage/prod?

Reply:
:facepalm: thanks!